addons/hr_work_entry_contract/models/hr_contract.py

```diff
@@ -379,7 +379,7 @@ def _cron_generate_missing_work_entries(self):
         # retrieve contracts for the current month
         today = fields.Date.today()
         start = today + relativedelta(day=1, hour=0)
-        stop = today + relativedelta(day=31, hour=23, minute=59, second=59)
+        stop = today + relativedelta(month=start.month + 1, day=31, hour=23, minute=59, second=59)
```

Comment:
This will break on december as month will be equal to 13.

Use ``months=1`` instead